src/Hashing/ArgonHashing.php

```diff
@@ -232,20 +232,7 @@ public function getThreads()
      */
     protected function algorithm()
     {
-        $algo = __config()->hashing->driver;
```

Comment:
Just return the current algorithm name because the `Argon2` hashes have been separated.